packages/python-google-compute-engine/google_compute_engine/metadata_watcher.py

```diff
@@ -177,11 +176,8 @@ def _HandleMetadataUpdate(
         if not isinstance(e, type(exception)):
           exception = e
           self.logger.error('GET request error retrieving metadata. %s.', e)
-        if retry:
-          retry_count += 1
-          continue
-        else:
-          break
+        time.sleep(1)
+        continue
```

Comment:
continue is not needed at end of loop. where are we decrementing the `retries` variable? won't this run in an infinite loop, now?

Reply:
Just fixed that all. Accidentally deleted.